flan/src/lib/MessageContents.svelte

```diff
@@ -1,23 +1,50 @@
 <script lang="ts">
 	import { MessageType, type IMessage } from './types';
 	import FileIcon from './svgs/File.svelte';
-	import { getModalStore, type ModalSettings } from '@skeletonlabs/skeleton';
+	import { getModalStore, getToastStore, type ModalSettings, type ToastSettings } from '@skeletonlabs/skeleton';
 
+	const toastStore = getToastStore();
 	const modalStore = getModalStore();
 	export let message: IMessage;
 	$: isLink = message.text?.startsWith('http');
+	$: isPassword = message.text?.includes('||');
+	$: messageSplit = isPassword && message.text?.split(' ');
 	$: modal = {
 		type: 'component',
 		component: 'imageModal',
-		meta: {image: message.payload?.src, name: message.payload?.name}
+		meta: { image: message.payload?.src, name: message.payload?.name }
 	} as ModalSettings;
+	function copySpoiler(e: MouseEvent) {
+		if (!e) return;
+		const wordElement = e.target as HTMLElement
+		navigator.clipboard.writeText(wordElement.innerText);
+		const toastMessage: ToastSettings = {
+			message: 'Copied to clipboard 🤓',
```

Comment:
"copied secret to clipboard" might serve better

Reply:
addressed